docs/vectorizer-api-reference.md

```diff
@@ -127,18 +127,18 @@ in other management functions.
 | Name             | Type                                                   | Default                           | Required | Description                                                                                        |
 |------------------|--------------------------------------------------------|-----------------------------------|----------|----------------------------------------------------------------------------------------------------|
 | source           | regclass                                               | -                                 | ✔        | The source table that embeddings are generated for.                                                |
-| destination      | name                                                   | -                                 | ✖        | A name for the table the embeddings are stored in.                                                 |
+| destination      | name                                                   | -                                 | ✖        | A name for the table the embeddings will be or are stored in. If defined, *view_name* argument takes this value, and the value of the *target_table* argument is set as `<destination>_store`.                                     |
```

Comment:
I find it confusing that we reference other arguments to explain how this works.

```suggestion
| destination      | name                                                   | -                                 | ✖        | Sets the name of the embedding table and view. Use this to avoid naming conflicts when configuring additional vectorizers for a source table. The view is named `<destination>`, and the embedding table is named `<destination>_store`.                                     |
```

Reply:
I felt we should clarify that, normally, you either define `destination` or `target_table` and `view_name`. How could we add this without naming those args?

Reply:
Set the name of table embeddings are stored in, and the view \<whatever it does>. The view is named `<destination>`, the embedding table is named `<destination>_store`. You set `destination` to avoid naming conflicts when you configure additional vectorizers for a source table.

Reply:
I ended up doing a mix of both of your suggestions. Please check 🙏